connect/runtime/src/test/java/org/apache/kafka/connect/runtime/isolation/PluginScannerTest.java

```diff
@@ -145,10 +148,28 @@ public void testScanningMixOfValidAndInvalidPlugins() throws Exception {
         assertEquals(expectedClasses, classes);
     }
 
+    @Test
+    public void testScannedPluingsForUndefinedVersion() {
+        PluginScanResult unversionedPluginsResult = scan(filterPluginsResourceDir(TestPlugins.pluginPath(), TestPlugins.TestPlugin.SAMPLING_HEADER_CONVERTER.resourceDir()));
```

Comment:
There's a utility method for assembling a plugin path of just select plugins, you don't need to do the filtering yourself :)
```suggestion
        PluginScanResult unversionedPluginsResult = scan(TestPlugins.pluginPath(TestPlugins.TestPlugin.SAMPLING_HEADER_CONVERTER));
```

Reply:
Thanks Manually committed to add TestPlugins dependency workaround